Articles/Blog/2024-01-zfp-sse.md

```diff
@@ -0,0 +1,107 @@
+
+**Mark:** Do you feel like ECP did its part by providing funding and deploying technologies and it was left up to you and your team to learn to use and apply them? Do you feel like ECP really supported you in your SSE quests on zfp?
+
+**Peter:** I can absolutely say that zfp would not be anywhere near where it is without the *funding* ECP provided.
+This allowed me to hire developers/software engineers to do the work to transform zfp from a research prototype to a production-quality package.
+I had to do a lot of learning on the job; other than funding, I can't point to anything in particular that ECP provided to facilitate this learning process, though I am aware that ECP has also supported various training activities (of which I, unfortunately, did not have the opportunity take advantage).
+Another thing ECP did was give us access to hardware vendor software engineers who helped with performance tuning.
+
+Working together with ECP Application Development (AD) and Software Technologies (ST) customers has also benefited zfp in terms of learning about features that would be useful and in terms of integrating zfp into other packages and applications.
+I found many ECP meetings to be productive because most of these interactions occurred at them.
+My institution, Lawrence Livermore National Laboratory (LLNL), provided additional support through their [RADIUSS](https://computing.llnl.gov/projects/radiuss) program, particularly to develop the Python API.
+There have been other funding sources for zfp, though primarily for basic or applied research on floating point data compression and not directly responsible for SSE.
+
+I also feel that LLNL's change in stance from supporting only restrictive, custom licenses (circa 2015) to encouraging open-source development under more permissive licenses has also helped a lot.
+I used to maintain zfp release assets in a somewhat adhoc fashion on the LLNL Computing department's web server.
+[Ian Lee](https://github.com/IanLee1521) took it upon himself to set up a GitHub repo for zfp (and other products), which really got the ball rolling on starting new SSE practices.
+
+**Mark:** If you had it to do all over again (I mean the SSE learning curve/ramp), how would you do it differently and/or what resources would have made it easier/faster?
+
+**Peter:** To be honest, I don't think I would have benefited as much from taking formal training to learn the tools and practices I now use daily.
+I feel that I need to get my hands dirty and *do* in order to learn.
+But it was very helpful to work with and learn from someone (Markus Salasoo, a zfp developer who had industry experience with SSE).
+I think I relied a bit too much on Markus in the beginning.
+It was only after he left the project that I really had to role up my own sleeves and get immersed in git/GitHub and learn how to use these and other tools.
```

Comment:
role up -> roll up

Reply:
Fixed in c9cda8b8a4678cc5efd79a19c8e75c13a58dfccd